application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2699,6 +2702,25 @@ public function update_response($sSessionKey, $iSurveyID, $aResponseData)
             return 'Error: Survey does not allow edit after completion.';
         }
 
+        if (count($aResponseData)==2 && isset($aResponseData['id']) && isset($aResponseData['token']))
```

Comment:
Totally different function then : best is to create a new function : `update_token_in_response($sSessionKey, $iSurveyID, $iResponseId,$sToken)`